client/my-sites/themes/theme-showcase.jsx

```diff
@@ -343,10 +338,11 @@ class ThemeShowcase extends Component {
 				.filter( ( key ) => ! subjectFilters.includes( key ) )
 				.join( '+' );
 
-			const newFilter =
-				tabFilter.key !== this.staticFilters.ALL.key
-					? [ filterWithoutSubjects, subjectTerm ].join( '+' )
-					: filterWithoutSubjects;
+			const newFilter = ! [ this.staticFilters.ALL.key, this.staticFilters.MYTHEMES.key ].includes(
```

Comment:
What if we replace this with something like `this.staticFilters.includes(tabFilter)` ? I find that the intention is clearer this way.

(I think we can do it because for static filters we always return the same object, CMIIW)

Reply:
Done. I made a `isStaticFilter` member function to check whether a filter is a static filter!